Documentation/dev-guide/limit.md

```diff
@@ -2,7 +2,7 @@
 
 ## Request size limit
 
-etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. For the time being, etcd guarantees to support RPC requests with up to 1MB of data. In the future, the size limit may be loosened or made configurable.
+etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. As of v3.2.12, etcd guarantees to support client RPC requests of 1.5 MiB, by default. The client size limit is configurable via `clientv3.Config.MaxCallSendMsgSize`, with corresponding `--max-request-bytes` flag for etcd server.
```

Comment:
etcd is designed to handle small key value pairs typical for metadata. Larger requests will work, but may increase the latency of other requests. By default, the maximum size of any request is 1.5 MiB. This limit is configurable through `--max-request-bytes` flag for etcd server.

Reply:
@gyuho if you want to add more documentation on the client side limit. submit a new PR for that and put it into a separate section. i do not want to mix server/client together.

Reply:
Ok sounds good. Will separate out.